src/CMF.jl

```diff
@@ -1,6 +1,7 @@
 module CMF
 
 # Dependendies.
+import PyPlot
```

Comment:
If this works, it might be better to have `const plt = PyPlot` right next to the import here, rather than in `visualize.jl`.